baseline-network-linux-hld.md

```diff
@@ -0,0 +1,163 @@
+# Unified Open Networking Hardware Support
+
+[[_TOC_]]
+
+## Scope
+
```

Comment:
I'd expect a document like this to lead with a definition of "open networking hardware".